### PR TITLE
Unify scalability 5k presubmit with postsubmit

### PR DESCRIPTION
**Pull request by wojtek-t**
/assign @marseel

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml [hunk 1]
```diff
@@ -222,7 +222,7 @@ presubmits:
     decorate: true
     path_alias: k8s.io/kubernetes
     decoration_config:
-      timeout: 570m
+      timeout: 450m
```
Comment (marseel) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:225]: Probably you need to remove `branches` above?

Comment (wojtek-t) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml:225]: No - I think this is right. It shows on what branches a given presubmit can be run.

**Pull request merged**